deprecated.tex

```diff
@@ -32,7 +32,6 @@
 The following algorithms are deemed unsafe and are deprecated in this document.
```

Comment:
Nitpick: let's change this to "algorithm signatures." It isn't the algorithms - e.g. `mismatch` - that are unsafe, it's these particular overloads.